README.md

```diff
@@ -139,7 +139,7 @@ To enable Kate RPC you need to pass `--enable-kate-rpc` flag when executing the
 ### Kate RPC
 ```bash
 ./avail-node --dev --enable-kate-rpc-metrics
-deno run -A ./examples/deno/benchmarks/query_proof.ts && deno run -A ./examples/deno/benchmarks/query_data_proof.ts && deno run -A ./examples/deno/benchmarks/query_rows.ts && deno run -A ./examples/deno/benchmarks/query_block_length.ts && deno run -A ./examples/deno/benchmarks/query_app_data.ts
+deno run -A ./examples/deno/benchmarks/query_proof.ts && deno run -A ./examples/deno/benchmarks/query_rows.ts && deno run -A ./examples/deno/benchmarks/query_block_length.ts && deno run -A ./examples/deno/benchmarks/query_app_data.ts
```

Comment:
You are missing `deno run -A ./examples/deno/benchmarks/query_data_proof_v2.ts`

Reply:
it was missing already